Editor/Unity.WebRTC.Editor.asmdef

```diff
@@ -11,5 +13,6 @@
     "precompiledReferences": [],
     "autoReferenced": true,
     "defineConstraints": [],
-    "versionDefines": []
+    "versionDefines": [],
+    "noEngineReferences": false
```

Comment:
It seems this property is not correct here.
```
No Engine References: When you enable this property, Unity does not add references to UnityEditor or UnityEngine when it compiles the assembly.
```
https://docs.unity3d.com/Manual/class-AssemblyDefinitionImporter.html